### DISCUSSION TITLE
BottomLightPWM_xxx and the LightPWM_xxx for Anycubic Photon Mono X

### DISCUSSION THREAD

**Discussion by robingantois** [sn4k3/UVtools#163] [Q&A] [answered]
Anycubuc photon Mono x; 
Why can't I manipulate the BottomLightPWM_xxx and the LightPWM_xxx?
The parameters do not appear in uvtools.
am I doing something wrong?

**Comment by sn4k3** [comment, marked as answer]
If it doesn't show for that format, that means it's not supported by the format itself.
Even the formats that support that, most don't respect nor use it anyway because LED is only ON or OFF and not controlable on most printers.

If you want to weaken LED power you must dim pixels instead or use less exposure as alternative

**Reply by robingantois** [reply]
I was recommended to only use the power of the photon mono x at 70% ... do I have to set that manually on the printer?

**Reply by sn4k3** [reply]
I don't know as i don't own any of that, but you can check, do a dummy print without vat and go to options while print. Check if there's an option to do such. Or maybe at printer options...

**Reply by robingantois** [reply]
Oops maybe I asked my question incorrectly.  on photon mono x you can set the power manually.  I had hoped that this could also be done via the software ...

**Reply by sn4k3** [reply]
AFAIK is not possible via software

**Reply by robingantois** [reply]
why then is there a possibility in prusaslicer profiles to adjust LightPWM_xxx?

**Reply by sn4k3** [reply]
Maybe i forgot to remove that unused variables when i was copying printers...

**Reply by sn4k3** [reply]
Here you can find a list of supported variables for the format which is also what uvtools shows under edit parameters: https://github.com/sn4k3/UVtools/blob/master/UVtools.Core/FileFormats/PhotonWorkshopFile.cs#L923

Commented lines (//) are to ignore
